docs/rpc/rpc-spans.md

```diff
@@ -52,37 +51,15 @@ This document defines how to describe remote procedure calls
 
 ### Span name
 
-The *span name* MUST be the full RPC method name formatted as:
+RPC spans MUST follow the overall [guidelines for span names](https://github.com/open-telemetry/opentelemetry-specification/blob/v1.51.0/specification/trace/api.md#span).
 
-```
-$package.$service/$method
-```
+RPC span names SHOULD be `{rpc.method.name}` if there is `rpc.method.name`
+available and it's not set to `_OTHER`.
+In other cases (when `{rpc.method.name}` is set to `_OTHER` or is not available),
+span name span name SHOULD be `{rpc.system.name}`.
 
-(where $service MUST NOT contain dots and $method MUST NOT contain slashes)
-
-If there is no package name or if it is unknown, the `$package.` part (including the period) is omitted.
-
-Examples of span names:
-
-- `grpc.test.EchoService/Echo`
-- `com.example.ExampleRmiService/exampleMethod`
-- `MyCalcService.Calculator/Add` reported by the server and
-  `MyServiceReference.ICalculator/Add` reported by the client for .NET WCF calls
-- `MyServiceWithNoPackage/theMethod`
-
-### Service name
```

Comment:
I don't think this section is helpful here now since we're removing `rpc.service` along with source of possible confusion with `service.name`.